core/src/avm1/globals/movie_clip.rs

```diff
@@ -794,15 +794,15 @@ fn get_bytes_loaded<'gc>(
     _activation: &mut Activation<'_, 'gc, '_>,
     _args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    Ok((movie_clip.movie().unwrap().data().len() + 20).into())
+    Ok((movie_clip.movie().unwrap().header().uncompressed_length).into())
```

Comment:
Let's do `movie_clip.movie().map(|movie| movie.header().uncompressed_length).unwrap_or_default()`, in case the movie happens to be None for some reason.

(This probably won't happen, and it was already an `unwrap`, but I try to avoid unwraps in cases like this now to fail more gracefully instead of panicking the entire player).

Reply:
Done.